nox/command.py

```diff
@@ -70,7 +75,7 @@ def run(
     paths: Optional[List[str]] = None,
     success_codes: Optional[Iterable[int]] = None,
     log: bool = True,
-    external: bool = False,
+    external: Union[Literal["error"], bool] = False,
```

Comment:
Thanks for catching this!

How about making this `Union[str, bool]` though? Being so specific here means tighter coupling, and forces us to pull in the dependency on `typing_extensions` for older Pythons.

To be clear, I'm very much in favor of using modern typing constructs, and I agree that we might pull in that dependency anyway in the future. I'm just not convinced we should be using `Literal` here specifically.

Reply:
On second thoughts, I'm fine with using `Literal` here.

It's nice to have mypy detect typos in string literals, and it's definitely an idiomatic use of `Literal` as the parameter can only ever be `False`, `True`, or `"error"`.